src/ServiceControl.UnitTests/ScatterGather/GetKnownEndpoints_ScatterGather_LocalInstanceReturnsNoData.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace ServiceControl.UnitTests.ScatterGather
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using NUnit.Framework;
+    using ServiceControl.CompositeViews.Endpoints;
+    using ServiceControl.CompositeViews.Messages;
+
+    [TestFixture]
+    class GetKnownEndpoints_ScatterGather_LocalInstanceReturnsNoData : GetKnownEndpoints_ScatterGatherTest
+    {
+        protected override IEnumerable<QueryResult<List<KnownEndpointsView>>> GetData()
+        {
+            yield return LocalPage(50);
+            yield return RemotePage(1);
+        }
+
+        [Test]
+        public void ResultsMatchRemote()
+        {
+            var remoteResults = RemotePage(1);
+            var returnedEndpointIds = Results.Results.Select(x => x.Id).ToList();
+
+            foreach (var remoteResult in remoteResults.Results)
```

Comment:
This test would pass when the list is empty. Should we fix that @mikeminutillo ?

Reply:
Good catch